src/Platform/Microsoft.Testing.Extensions.HotReload/HotReloadHandler.cs

```diff
@@ -51,6 +51,7 @@ private static bool IsCancelKeyPressNotSupported()
         => RuntimeInformation.IsOSPlatform(OSPlatform.Create("ANDROID")) ||
             RuntimeInformation.IsOSPlatform(OSPlatform.Create("IOS")) ||
             RuntimeInformation.IsOSPlatform(OSPlatform.Create("TVOS")) ||
+            RuntimeInformation.IsOSPlatform(OSPlatform.Create("WASI")) ||
```

Comment:
Are you sure the keyword is `WASI` and not `WASM`? I have no idea honestly, just asking.

Reply:
Based on .NET runtime code, I think yes it's WASI.

https://github.com/dotnet/runtime/blob/15872212c29cecc8d82da4548c3060f2614665f7/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RuntimeInformation.cs#L26

https://github.com/dotnet/runtime/blob/15872212c29cecc8d82da4548c3060f2614665f7/src/libraries/System.Private.CoreLib/src/System/OperatingSystem.cs#L127

https://github.com/dotnet/runtime/blob/15872212c29cecc8d82da4548c3060f2614665f7/src/libraries/System.Private.CoreLib/src/System/OperatingSystem.cs#L14-L43

Reply:
Oh yeah, makes sense, it's `wasi` for `wasi-wasm` and `browser` for `browser-wasm`.